src/task-runtime.ts

```diff
@@ -87,7 +88,8 @@ class RunTask {
     private readonly runtime: TaskRuntime,
     private readonly task: TaskSpec,
     parents: string[] = [],
-    args: Array<string | number> = []
+    args: Array<string | number> = [],
+    envParam: { [name: string]: string } = {}
```

Comment:
These end up as keyword args in python, so let's make sure we are using a "good" name:
```suggestion
    env: { [name: string]: string } = {}
```